resources/control_systems.md

```diff
@@ -5,66 +5,712 @@
+| $\omega_pc = \omega_gc$ | Marginally Stable system |
+| $\omega_pc < \omega_gc$ | Unstable system          |
+
+##### 14.1.1.2 Gain and Phase Margin
+
+-**Gain Margin**, GM: This is the negative value of magnitude in dB at the phase cross over frequency, $M_pc$.
+
+It is mathematically defined as
+
+$$GM = 20\log(\frac{1}{M_pc}) = 10\log(M_pc)$$
+
+- **Phase Margin**, PM: It is mathematically defined as
+
+$$PM = 180^\circ + \phi_gc$$
+
+where $\phi_gc$, is the phase angle at the gain cross over frequency.
+
+| Stability Criteria            | Stability                |
+|-------------------------------|--------------------------|
+| $GM > 0$<br/> and $PM > 0$    | Stable system            |
+| $GM = 0$<br/> and $PM = 0$    | Marginally Stable system |
+| $GM < 0$<br/> and/or $PM < 0$ | Unstable system          |
+
+**Recommended video:**
+> [How to find cross over frequencies, Gain and Phase Margin from Bode Plot](https://www.youtube.com/watch?v=iB10Hk0BCAs)
+
+#### 14.1.2 MATLAB Implementation
+
+**Recommended video:**
 > [Bode plot of frequency response, or magnitude and phase data](https://www.mathworks.com/help/ident/ref/dynamicsystem.bode.html)
 
-### 6.2 Nichols Chart
+### 14.2 Nyquist Plot
+
+#### 14.2.1 Nyquist Stability Criterion
+
+The criterion states that if _P_ poles and _Z_ zeros are encircled by an enclosed path in the _s_ plane,
+the corresponding _G(s)H(s)_ plane must encircle the origin _P-Z_ times. The encirclement number, _N_,
+is given by
+
+$$N = P - Z$$
+
+- If only poles exist in the encircled _s_ plane closed path, the encirclement in the _G(s)H(s)_ is
+  opposite in direction to the path in the _s_ plane.
+- If only zeros exist in the encircled _s_ plane closed path, the encirclement in the _G(s)H(s)_ is
+  in the same direction as the path in the _s_ plane.
+
+The **Nyquist Stability Criterion** states that _N_ of the point (-1,0) must equal _P-Z_ of the close loop
+transfer function.
+
+**Recommended video:**
+> [Nyquist plot intro 1](https://www.youtube.com/watch?v=sof3meN96MA)
+
+**Recommended video:**
+> [Nyquist plot intro 2](https://www.youtube.com/watch?v=tsgOstfoNhk)
+
+#### 14.2.2 Stability Analysis with Nyquist Plots
+
+##### 14.2.1.1 Cross Over Frequency
+
+- **Phase Cross Over Frequency**, $\omega_pc$: This is the frequency at which the Nyquist plot
+  intersects the negative real axis (phase of $-180^\circ$). Its unit is **rad/sec**.
+- **Gain Cross Over Frequency**, $\omega_gc$: This is the frequency at which the Nyquist
+  plot has a magnitude of 1. Its unit is **rad/sec**.
+
+| Stability Criteria      | Stability                |
+|-------------------------|--------------------------|
+| $\omega_pc > \omega_gc$ | Stable system            |
+| $\omega_pc = \omega_gc$ | Marginally Stable system |
+| $\omega_pc < \omega_gc$ | Unstable system          |
+
+##### 14.2.1.2 Gain and Phase Margin
+
+-**Gain Margin**, GM: This is the reciprocal value of magnitude at the phase cross over frequency, $M_pc$.
```

Comment:
Minor mistake on point form `-Some Text` resulting in no proper markdown rendering.